### DISCUSSION TITLE
Multi-account support

### DISCUSSION THREAD

**Discussion by sy-records** [community/community#73365] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

~What is the purpose of the Add account here?~ https://github.blog/changelog/2023-11-03-multi-account-support-on-github-com/

After using account password, I was redirected to https://github.com/sessions/verified-device/mobile?auto=true and got a 404 message. 

I also got a `please verify your device` email. Manually visit https://github.com/sessions/verified-device and enter the captcha code and enter successfully

**Comment by meeg** [comment]
I was having the identical problem, here is more detail and possibly a workaround.

To start, I'm in a browser on my PC in Chrome, I'm logged into my main account (@meeg), I'm trying to log in to a service account (@qickbot), I have GitHub Mobile 2FA on both accounts. I tried this on computers where I've never logged into the service account (so I try to "Add account"), and computers where it's in the account switcher, but the login is expired (so I click "See all accounts," see that I'm signed out of the service account, and click the "Sign in" button). Either way, I enter my password, and get sent to https://github.com/sessions/verified-device/mobile?auto=true and a 404. I do not get a 2FA login request on GitHub Mobile.

I stubbornly tried this same thing over a few days but eventually gave up to do the obvious thing; I logged out of my main account and logged into the service account, which worked. I then tried adding the main account to the account switcher. Surprisingly, this also worked (and I get 2FA requests for both on GitHub Mobile). I go to another computer which is logged in to the main account (where I was previously unable to add the service account), and am able to add the service account.

So I don't know what's going on behind the scenes, but it seems like logging in to the second account outside of the multi-account flow "unsticks" something in the system?

**Comment by prakharbirla-ng** [comment]
Surprisingly, I'm still facing the same issue..

**Comment by ConnorDoesDev** [comment]
Same here. Facing the same issue on Brave

**Comment by farazalidev** [comment]
same issue :(

**Comment by hosamn** [comment]
Same here :(

**Comment by prakharbirla-ng** [comment]
It's so odd the this issue isn't getting any fixes. It's an accessibility issue which is preventing users from using your product.

**Comment by Alec269** [comment]
same issue. though i only have MFA on my main account. When will this get fixed?

**Comment by idata-frank** [comment]
Same issue, though I can swap accounts on the github mobile app.

**Reply by stomper0** [reply]
Following up on this, I managed to get it to work by deleting `/mobile?auto=true` from the URL you get redirected to, the web page then sends an MFA code that I can use to log in.

For example, the URL I am redirected to is
`https://github.com/sessions/verified-device/mobile?auto=true`

I delete the end, and go to
`https://github.com/sessions/verified-device`

in my browser, and this allows me to add an account without signing out of the other.

**Comment by iharee** [comment]
Same issue. Has it really not been resolved even now?
